Trimble.Modus.Components/Controls/Popup/Pages/PopupPage.cs

```diff
@@ -320,12 +320,12 @@ private void OnSizeChanged(object sender, EventArgs e)
                 {
                     case Enums.ModalPosition.Top:
                         translationY = loc.Top - _popupHeight - loc.Height / 2;
-                        translationX = loc.Center.X - _popupWidth / 2;
+                        translationX = loc.X;
                         break;
 
                     case Enums.ModalPosition.Bottom:
                         translationY = loc.Bottom - _popupHeight + loc.Height / 2;
-                        translationX = loc.Center.X - _popupWidth / 2;
+                        translationX = loc.X;
```

Comment:
Can you please add the reason why we changing this with loc.x as comment

Reply:
Fixed